snip-gui/src/app/Components/admin-view/admin-view.component.html

```diff
@@ -46,11 +46,11 @@ <h4>Admin Page</h4>
     </div>
     <!-- Remove old snippet functionality -->
     <div class="Row">
-        <div class="col-lg-5 border bg-dark text-white" style="height">
+        <div class="col-lg-5 border bg-dark text-white" style="height: 250px;">
```

Comment:
I'm moving a lot of styling into the .css file to make it more accessible. This is fine for now, but eventually we should move it all there.

Reply:
agreed